selection/selection.py

```diff
@@ -40,20 +35,23 @@ def select(self):
             TrainingSet
         """
 
-        # inspect some of the training data
-        print(self.target_vectors.shape)
-        print(self.nontarget_mswc_vectors.shape)
-        print(len(self.target_ids), self.target_ids[0])
-        print(len(self.nontarget_ids), self.nontarget_ids[0])
+        if self.audio_flag:
+            print(self.target_vectors['non_targets'][0]['audio'])
+
+        per_class_size = self.train_set_size // 6 #5 targets + nontarget
+
+        selected_targets = {}
+        for target, sample_list in self.target_vectors['targets'].items():
+            selected_samples = sample(sample_list, per_class_size)
+            selected_targets[target] = [sample['ID'] for sample in selected_samples]
 
-        rng = np.random.RandomState(0)
+        selected_nontarget_samples = sample(self.target_vectors['non_targets'], per_class_size)
```

Comment:
I think we will want to add in the random seed from the config and pass that in as a constructor argument, so we might as well use numpy here (especially to encourage future participants to be as deterministic as possible for repeatability)